.circleci/config.yml

```diff
@@ -90,21 +84,42 @@ jobs:
             bundle exec rake knapsack:rspec
             yarn test
             bundle exec slim-lint app/views
-
       - run:
           name: Code Climate Test Coverage
           command: |
             ./cc-test-reporter format-coverage -t simplecov -o "coverage/codeclimate.$CIRCLE_NODE_INDEX.json"
             aws s3 sync coverage/ "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM"
-
       # collect reports
       - store_test_results:
           path: /tmp/test-results
       - store_artifacts:
           path: /tmp/test-results
           destination: test-results
-
       - deploy:
           command: |
             aws s3 sync "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM" coverage/
             ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --input -
+  build-container:
+    working_directory: ~/identity-idp
+    docker:
+      - image: circleci/ruby:2.6
+    steps:
+      - setup_remote_docker
+      - run: |
+          docker build -t logindotgov/idp:$CIRCLE_TAG -f production.Dockerfile .
+          echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
```

Comment:
We'll need to get these added to the ENV variables in CirlceCI, will we just create some service user in DockerHub?

Reply:
Yup, I'm going to be working on setting that up today, if I can.

Reply:
This is fixed, just waiting on #3730 to be released, then I'll rebase.